pyproject.toml

```diff
@@ -34,7 +34,8 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 ignore = ["W503", "E501", "D107"]
 # D100, D101, D102, D103: Ignore missing docstrings in tests
 per-file-ignores = [
-    "tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DCO020,DCO030,DCO050"
+    "tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DCO020,DCO030,DCO050",
+    "xiilib/_gunicorn/charm.py:DCO060"
```

Comment:
What is the reason for not writing a docstring please? The comment above notes that the current disable is for the tests which doesn't seem to be the case here

Reply:
I think there's a problem with `typing.Protocol`?

Reply:
What is the issue please?

Reply:
Okay, I removed the ignore and it seems like it's not the `typing.Protocol`'s issue since I didn't include `Returns` in the abstract methods' docstring. This is cause by property function's need to be included in the class `Attrs`. I will updated this, thanks!

Reply:
removed